src/components/Footer/index.tsx

```diff
@@ -32,6 +32,9 @@ export const Footer = () => {
                 <a href={'https://dothq.link/dsc'} target={'_blank'}>
                   <i className={'discord-icon'}></i>
                 </a>
+                <a href={'https://dothq.link/matrix'} target={'_blank'}>
+                <i className={'matrix-icon'}></i>
+              </a>
```

Comment:
This too please.

Reply:
ok

Reply:
I will open up a new pull request with the requested changes

Reply:
Why? Just make your changes to this one.